docs/installation.md

```diff
@@ -244,3 +244,51 @@ Kubernetes updates the mounted file when the ConfigMap is updated.
 Standard Helm oœptions such as `replicaCount`, `image`, `imagePullSecrets`, 
 `service`, `ingress` and `resources` are supported. These are defined in 
 `helm/values.yaml`. 
+
+### Health Checks
+
+Trino Gateway checks the health of each backend and deactivates it if unhealthy.
```

Comment:
I'd probably bold "deactivates it if unhealthy" because it's an important behavior and sometimes confuses people why their backends are deactivated.